### ISSUE TITLE
Remove installer.vm and deboat.vm from package customization GUI

### ISSUE THREAD

**Issue by emtuls** [mandiant/flare-vm#546]
### Details

`installer.vm` and `debloat.vm` are both automatically installed during the `flare-vm` installation inside of `install.ps1`, so they are not necessary to be displayed inside of the GUI interface for package options to select from.

**Comment by Ana06** [comment]
I agree, we should remove then from the installer GUI. Would you like to address this @emtuls?

**Comment by Ana06** [comment]
This issue can be addressed together with https://github.com/mandiant/flare-vm/issues/578
